bin/triton-bootstrap

```diff
@@ -194,7 +205,7 @@ else
     while [ $CLUSTERFOUND != 1 ]; do
         echo -n '.'
 
-        CLUSTERIP=$(curl -L -s -f http://consul:8500/v1/catalog/service/couchbase | json -aH ServiceAddress | head -1)
+        CLUSTERIP=$(curl -L -s -f curl http://consul:8500/v1/health/service/couchbase?passing | json -aH Service.Address | head -1)
```

Comment:
@dekobon: I think you were looking for this endpoint previously. This is the way to query Consul and _only_ get back healthy services.